src/themes/default/css/vendor/_flexboxgrid2.scss

```diff
@@ -83,6 +83,40 @@
   max-width: 100%;
 }
 
+.col-xs,
+.col-xs-1,
```

Comment:
probably could be simplified to 
```
[class^="col-xs"], [class*=" col-xs"] {
...
}
```

Reply:
@lukeromanowicz I think  below is better its works for both case 
```
[class*=" col-xs"] {
...
}
```

No need to write
`[class^="col-xs"]`

Reply:
Actually, there is a difference in matching. Please, take a look at https://jsfiddle.net/ycrLtgmf/

Reply:
@lukeromanowicz 
Yes, Got it difference.
Let me a simplified code as per your suggestion.
Thanks.